projects/core/components/scrollbar/scrollbar.component.ts

```diff
@@ -51,10 +51,10 @@ export class TuiScrollbar {
     protected readonly browserScrollRef = new ElementRef(this.el);
 
     @Input()
-    public hidden = false;
+    public hidden = this.options.mode === 'hidden';
```

Comment:
Should we deprecate it and instead make a `mode` input?

Reply:
fixed